firefox-ios/Client/Frontend/Autofill/Address/Edit/EditAddressViewController.swift

```diff
@@ -38,50 +40,42 @@ class EditAddressViewController: UIViewController, WKNavigationDelegate, WKScrip
         fatalError("init(coder:) has not been implemented")
     }
 
-    override func loadView() {
-        view = webView
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
         setupWebView()
+        setupActivityIndicator()
+
         listenForThemeChange(view)
     }
 
+    private func setupActivityIndicator() {
+        activityIndicator = UIActivityIndicatorView(style: .large)
+        activityIndicator.hidesWhenStopped = true
+        activityIndicator.translatesAutoresizingMaskIntoConstraints = false
+        self.view.addSubview(activityIndicator)
+        NSLayoutConstraint.activate([
+            activityIndicator.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            activityIndicator.centerYAnchor.constraint(equalTo: view.centerYAnchor)
+        ])
+    }
+
     private func setupWebView() {
+        webView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
```suggestion
        webView.translatesAutoresizingMaskIntoConstraints = false
```
Nit: This could be part of the initial initialization